### ISSUE TITLE
do you have objection detection .distiller demo?

### ISSUE THREAD

**Issue by hexiangquan** [irfanICMLL/structure_knowledge_distillation#16]

**Comment by irfanICMLL** [comment]
You can use the demo code in FCOS. Our model can be directly loaded into the FCOS demo. If you have any questions, feel free to contact me.

**Comment by anhvth** [comment]
hi irfan, could you provide the training code for fcos distilation

**Comment by irfanICMLL** [comment]
We do not have the plan for releasing fcos distilation recently. I use the training code of FCOS and insert the loss function of distillation. It is not very hard to reproduce. If you have any questions, feel free to contact me.
